### PR TITLE
Feature/loading

### PR DESCRIPTION
**Pull request by snackthyme**
Our website no longer looks like it's doing nothing for several seconds while Heroku gets its shit together

Might need some tweaks for remote

### CODE REVIEW

File: FoundBook/settings.py [hunk 1]
```diff
@@ -133,11 +133,13 @@
 PROJECT_ROOT = os.path.dirname(os.path.abspath(__file__))
 
 STATIC_URL = '/static/'
-STATIC_ROOT = os.path.join(PROJECT_ROOT, 'staticfiles')
```
Comment (cujomalainey) [inline FoundBook/settings.py]: are you sure this is correct? isn't this supposed to point to the non-compressed files?

File: FoundBook/settings.py [hunk 2]
```diff
@@ -133,11 +133,13 @@
...
+STATIC_ROOT = os.path.join(PROJECT_ROOT, 'static')
 
 # Extra places for collectstatic to find static files.
 STATICFILES_DIRS = (
-    os.path.join(PROJECT_ROOT, 'static'),
+    ('vendor', os.path.join(BASE_DIR, 'node_modules', 'd3', 'build')),
```
Comment (cujomalainey) [inline FoundBook/settings.py]: stuff for loading static content in parallel?

File: FoundBook/settings.py (hunk 1 above)
Comment (snackthyme) [inline FoundBook/settings.py]: STATIC_ROOT points to the actual location you want collectstatic to put the static files it finds. STATIC_URL is the URL you point your browser to in order to get to STATIC_ROOT on the backend.

File: FoundBook/settings.py (hunk 2 above)
Comment (snackthyme) [inline FoundBook/settings.py]: We weren't actually doing any automatic static file collection before, although the code was there. I'm using it to collect node-modules into /static/vendor/ so I can serve them.

Comment (snackthyme) [inline FoundBook/settings.py]: So in actual fact after this merge everyone will want to run `npm install` followed by `python manage.py collectstatic`

File: FoundBook/settings.py (hunk 1 above)
Comment (cujomalainey) [inline FoundBook/settings.py]: ok, ill verify it deploys then

**Pull request merged**